addons/mail/static/tests/messaging/messaging.test.js

```diff
@@ -88,6 +97,9 @@ test("Show conversations with new message in chat hub (outside of discuss app)",
     );
     await contains(".o-mail-ChatBubble[name='GroupChat']");
     await openDiscuss();
+    env.services.bus_service.subscribe("discuss.channel/new_message", () =>
+        expect.step("discuss.channel/new_message")
+    );
```

Comment:
I would intercept all of them at beginning of test and `expect.waitForSteps()` each of them.
While the subscribe is made after `openDiscuss()`, nothing prevents it to intercept the old `new_message` notifications by mistake.

The only safe-guard as it is with current code, is saying that chat bubble pops from necessarily `new_message` necessarily. It's not made obvious in the test right now, whereas if you have intercept at the very top and each chat bubble await `new_message` then the test is more confidence on `new_message` and pop of chat bubble working together.